### ISSUE TITLE
getTab wrong type

### ISSUE THREAD

**Issue by beig** [brrd/electron-tabs#69]
Using    "electron-tabs": "^0.9.4",

the function

`  getTab(id: string): ElectronTabs.Tab | null;`

is not working correctly because of type equality, this.tabs[id].id returns an integer

```
     getTab (id) {
        for (let i in this.tabs) {
          if (this.tabs[i].id === id) {
                return this.tabs[i];
            }
        }
        return null;
    }
```

**Comment by beig** [comment]
Sorry me neither,

I just tried it again and it works without changing anything.

**Comment by brrd** [comment]
Fixed with eeeebd41ce6a6fc6a837b33749b057da3c248314
